### ISSUE TITLE
Bugs in using the new interface view 

### ISSUE THREAD

**Issue by nicgef** [centreon/centreon#9937]
# BUG REPORT INFORMATION

### Prerequisites

> The opened issue, must be code related. GitHub is not meant for support. Feel free to check the CONTRIBUTING section for more details.

***Versions***

For the RPM based systems

-- Copy/Paste the result of the following command --
```
centreon-plugin-Applications-Protocol-Dns-20210622-151444.el7.centos.noarch
centreon-broker-storage-21.04.1-1.el7.centos.x86_64
centreon-engine-21.04.2-3.el7.centos.x86_64
centreon-license-manager-common-21.04.0-1.el7.centos.noarch
centreon-base-config-centreon-engine-21.04.2-15.el7.centos.noarch
centreon-widget-grid-map-21.04.0-1.el7.centos.noarch
centreon-plugin-Applications-Protocol-Ldap-20210622-151444.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Poller-20210622-151444.el7.centos.noarch
centreon-poller-centreon-engine-21.04.2-15.el7.centos.noarch
centreon-widget-service-monitoring-21.04.0-1.el7.centos.noarch
centreon-widget-host-monitoring-21.04.0-2.el7.centos.noarch
centreon-plugin-Applications-Protocol-Ftp-20210622-151444.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Map4-Jmx-20210622-151444.el7.centos.noarch
centreon-plugin-Hardware-Ups-Standard-Rfc1628-Snmp-20210622-151444.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Central-20210622-151444.el7.centos.noarch
centreon-plugin-Operatingsystems-Windows-Snmp-20210622-151444.el7.centos.noarch
centreon-clib-21.04.1-1.el7.centos.x86_64
centreon-perl-libs-21.04.2-15.el7.centos.noarch
centreon-trap-21.04.2-15.el7.centos.noarch
centreon-connector-perl-21.04.1-2.el7.centos.x86_64
centreon-gorgone-centreon-config-21.04.1-1.el7.centos.noarch
centreon-pp-manager-21.04.0-1.el7.centos.noarch
centreon-widget-global-health-21.04.0-1.el7.centos.noarch
centreon-widget-httploader-21.04.0-3.el7.centos.noarch
centreon-widget-hostgroup-monitoring-21.04.0-1.el7.centos.noarch
centreon-21.04.2-15.el7.centos.noarch
centreon-plugin-Applications-Protocol-Http-20210622-151444.el7.centos.noarch
centreon-plugin-Applications-Databases-Mysql-20210622-151444.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Database-20210622-151444.el7.centos.noarch
centreon-common-21.04.2-15.el7.centos.noarch
centreon-engine-daemon-21.04.2-3.el7.centos.x86_64
centreon-broker-core-21.04.1-1.el7.centos.x86_64
centreon-broker-cbd-21.04.1-1.el7.centos.x86_64
centreon-broker-cbmod-21.04.1-1.el7.centos.x86_64
centreon-connector-ssh-21.04.1-2.el7.centos.x86_64
centreon-gorgone-21.04.1-1.el7.centos.noarch
centreon-license-manager-21.04.0-1.el7.centos.noarch
centreon-widget-graph-monitoring-21.04.0-1.el7.centos.noarch
centreon-widget-tactical-overview-21.04.0-1.el7.centos.noarch
centreon-widget-engine-status-21.04.0-1.el7.centos.noarch
centreon-database-21.04.2-15.el7.centos.noarch
centreon-plugin-Network-Cisco-Standard-Snmp-20210622-151444.el7.centos.noarch
centreon-plugin-Operatingsystems-Linux-Snmp-20210622-151444.el7.centos.noarch
centreon-release-21.04-4.el7.centos.noarch
centreon-web-21.04.2-15.el7.centos.noarch
centreon-widget-live-top10-memory-usage-21.04.0-1.el7.centos.noarch
centreon-widget-servicegroup-monitoring-21.04.0-1.el7.centos.noarch
centreon-plugin-Hardware-Printers-Generic-Snmp-20210622-151444.el7.centos.noarch
centreon-broker-21.04.1-1.el7.centos.x86_64
centreon-engine-extcommands-21.04.2-3.el7.centos.x86_64
centreon-connector-21.04.1-2.el7.centos.x86_64
centreon-auto-discovery-server-21.04.0-1.el7.centos.noarch
centreon-widget-live-top10-cpu-usage-21.04.0-1.el7.centos.noarch
```

***Operating System***

cat /proc/version : Linux version 3.10.0-957.1.3.el7.x86_64 (mockbuild@kbuilder.bsys.centos.org) (gcc version 4.8.5 20150623 (Red Hat 4.8.5-36) (GCC) ) #1 SMP Thu Nov 29 14:49:43 UTC 2018

***Browser used***

- [X] Google Chrome
- [X] Firefox
- [X] Internet Explorer IE11
- [ ] Safari

Version: last

Server running on a VirtualBox virtual machine

### Description

On the new Centreon interface, I encountered many bugs related to the display of hosts and services. First of all, not all host groups appear in the drop-down menu: the first ten are displayed, not the following ones. If I search for a host group manually by typing it on the keyboard, I find it.
And secondly, when I display the hosts of a group and click on "Parent" to sort them, my filters disappear! 

### Steps to Reproduce

Please describe precisely the steps to reproduce the encountered issue.

1. I logged in Centreon
2. I go to the Monitoring > Resources Status menu 
3. In the filters, I click on the drop-down menu "Host Group" 

### Describe the received result

I only see the first 10 host groups, and I don't have a slider to go down to see all the others.
And when I select a hostgroup to filter my hosts, then sort them by name, the filters clear. 

### Describe the expected result

Being able to select any host group from the filters of the new interface, and sort the hosts by name / ip / etc. without the filters disappearing. 
![Screenshot_1](https://user-images.githubusercontent.com/74780741/123132470-12550a00-d44f-11eb-9afa-128185a4d9a7.png)

**Comment by sbrizini** [comment]
Hi @nicgef,

This is our first encounter with this, we're looking into it.

Thx,

**Comment by tuntoja** [comment]
Hello @nicgef 
We tried reproducing this on demo.centreon.com without success so far.

a few questions about the issues described

- do you log in as admin or as a non admin user?
- do you get any errors in the browser console (mainly network and console tabs) when using the Hostgroup drop down menu?
-

**Comment by nicgef** [comment]
Hello,

I log in with an administrator account.

Here is what is displayed to me in the Network tab, limit = 10.

![image](https://user-images.githubusercontent.com/74780741/123251671-c5734100-d4eb-11eb-92e2-c7318c1ec0fe.png)

**Comment by tuntoja** [comment]
Hello,
the issue with only 10 hostgroups being displayed from the Resources Status criteria drop down menuhas been reproduced and a ticket has been opened. 
Regarding the othe rissue about the sort by Parent causing the filters to disappear did you mean all filters, either search bar and/or any criteria selected? This one has not been reproduced yet on our platform.

**Comment by tuntoja** [comment]
The issue related to sorting has been reproduced as well. A ticket has been opened too.

**Comment by lpinsivy** [comment]
Resolved with Centreon Web 21.04.4
